OpenProblemLibrary/ma117DB/set5/srw2_9_63.pg

```diff
@@ -20,35 +20,37 @@ DOCUMENT();        # This should be the first executable line in the problem.
 
 loadMacros(
   "PGstandard.pl",
-  "PGchoicemacros.pl",
+  "PGML.pl",
+  "MathObjects.pl",
   "PGcourse.pl"
 );
 
-TEXT(beginproblem());
 $showPartialCorrectAnswers = 1;
 
+Context("Numeric");
+
 $a = random(2,9,1);
+$ans1 = Compute("-$a");
+$ans2 = Infinity;
+$ans3 = Formula("sqrt(x)-$a");
+
+BEGIN_PGML
+The function [`f(x)=(x+[$a])^2`] is not one-to-one. Choose a largest possible
+domain containing the number 100 so that the function restricted to the domain is one-to-one.
+
+The largest possible domain is [`\left[\strut\right.`][________]{$ans1},[________]{$ans2}[`\left.\strut\right)`].
```

Comment:
This would be cleaner using the `Interval` context.

Reply:
I was thinking about it, but did not want to make too large changes to the problem.

Reply:
IMO there is no such thing as too large of changes.  If there are places where the problem can be improved while still effectively asking the same question then I say go for it.